docs/includes/apiargs-MongoDBDatabase-method-modifyCollection-param.yaml

```diff
@@ -0,0 +1,19 @@
+source:
+  file: apiargs-common-param.yaml
+  ref: $collectionName
+replacement:
+  action: " or view to modify"
```

Comment:
In the interest of not overloading the purpose of `{{action}}`, can you see if we can change the `$collectionName` definition in `apiargs-common-param.yaml` to:

```yaml
arg_name: param
name: $collectionName
type: string
description: |
  The name of the {{subject}}{{action}}.
interface: phpmethod
operation: ~
optional: false
replacement:
  subject: "collection"
  action: ""
```

And then this can use the following replacement:

```yaml
replacement:
  subject: "collection or view"
  action: " to modify"
```